tests/integ/scala/test_dataframe_suite.py

```diff
@@ -3227,3 +3228,56 @@ def test_limit(session):
 |     |     |
 -------------\n""".lstrip()
     )
+
+
+@pytest.mark.skipif(
+    "config.getoption('local_testing_mode', default=False)",
+    reason="FEAT: session.sql is not supported and interval is in PrPr",
+)
+def test_yearmonth_interval_type_dataframe(session):
+    session.sql("alter session set feature_interval_types=enabled;").collect()
```

Comment:
you need to skip it in the stored proc test 

```
@pytest.mark.skipif(
    IS_IN_STORED_PROC,
    reason=...,
)
```
alter session is not supported in stored proc, same for all other places calling alter session